src/daemon.py

```diff
@@ -1,24 +1,23 @@
 #!/usr/bin/env python
-import json
 import logging
 import time
 import yara  # type: ignore
 from functools import lru_cache
-import random
 from yara import SyntaxError
 import config
 from lib.ursadb import UrsaDb
 from lib.yaraparse import parse_yara, combine_rules
-from util import make_redis, setup_logging
-from typing import Any, Dict, List, Optional, Tuple
+from util import setup_logging
+from typing import Any, Dict, List
+from db import Database, JobId, MatchInfo
 
-redis = make_redis()
+redis = Database()
```

Comment:
nitpicking again, but won't the `redis` variable name get confused with the redis connection?

Reply:
Actually, the code is not supposed to use redis connection directly from this class. So it shouldn't get confused :thinking: Does it make sense?

Reply:
I'm also speaking from my own experience, at the first read-through of the changes I was surprised that the redis connection has so many interesting methods.

Reply:
:thinking: You're probably right (also the database doesn't *have* to be redis in the future, its an implementation defaul at this point). It's just, all the good names are already taken. I'll think of something and refactor.